IGraphics/Platforms/IGraphicsWin.cpp

```diff
@@ -937,15 +932,6 @@ void IGraphicsWin::PlatformResize(bool parentHasResized)
   }
 }
 
-#ifdef IGRAPHICS_GL
```

Comment:
Is there no longer a need to activate and deactivate around draw resizing or is it happening some other way?

Reply:
It happens in NanoVG DrawResize() but i see that we also need it in Skia DrawResize

Reply:
I added it to skia